flac.md

```diff
@@ -630,6 +630,51 @@ Decoding the coded residual thus involves selecting the right coding method, fin
 
 Following the last subframe is the frame footer. If the last subframe is not byte aligned (i.e. the bits required to store all subframes put together are not divisible by 8), zero bits are added until byte alignment is reached. Following this is a 16-bit CRC, initialized with 0, with polynomial x^16 + x^15 + x^2 + x^0. This CRC covers the whole frame excluding the 16-bit CRC, including the sync code.
 
+# Container mappings
+
+The FLAC format can be used without any container as the FLAC format already provides for a very thin transport layer. However, the functionalty of this transport is rather limited, and to be able to combine FLAC audio with video, it needs to be encapsulated by a more capable container. This presents a problem: the transport layer provided by the FLAC format mixes data that belongs to the encoded data (like blocksize and sample rate) with data that belongs to the transport (like checksum and timecode). The choice was made to encapsulate FLAC frames as they are, which means some data will be duplicated and potentially deviating between the FLAC frames and the encapsulating container.
+
+The sample rate, bit depth and number of channels MUST be the same between all frame headers, the streaminfo metadata block and (if applicable) container metadata. In case an audio stream is encoded where these parameters change at some point in the stream, this should be dealt with as specified by the container.
```

Comment:
It should be a restriction per container format, e.g. MPEG-TS can support theses changes without problem. I see that it is already documented for Matroska, so this § may be deleted and just adding a tip about that for OGG.